services/spar/src/Spar/App.hs

```diff
@@ -105,7 +115,7 @@ import Wire.API.User.Scim (ValidExternalId (..))
 newtype Spar r a = Spar {fromSpar :: Member (Final IO) r => ReaderT Env (ExceptT SparError (Sem r)) a}
   deriving (Functor)
 
-liftSem :: (Member (Final IO) r => Sem r a) -> Spar r a
+liftSem :: Sem r a -> Spar r a
```

Comment:
Changing the type here because it confuses GHC and sometimes complains about `a` being untouchable at use-sites.

Reply:
:+1: